WORKING_GROUPS.md

```diff
@@ -287,6 +288,18 @@ Its responsibilities are:
 * Creating and operating a process for documentation review that produces
   quality documentation and avoids impeding the progress of Core work.
 
+### [Testing](https://github.com/nodejs/testing)
+
+The Node.js Testing Working Group's purpose is to extend and improve testing of
+the Node.js source code.
+
+It's responsibilities are:
```

Comment:
Has code coverage ever attempted on our source code? If it isn't, will it fall under the scope of this WG?